action.yml

```diff
@@ -78,6 +78,9 @@ inputs:
     required: false
   remote_states:
     description: Remote state blocks to configure in the workspace
+  team_access:
+    description: Teams and their associated permissions to be granted to the created workspaces
```

Comment:
Document the type of objects expected in these descriptions. And that they're YAML.